src/converter/dimacs/LogicalExpressionParser.ts

```diff
@@ -5,72 +5,42 @@ import {
   variableRule,
   blankRule,
 } from "./Syntax/CommonSyntax";
-import {
-  andRule,
-  negatedVariableRule,
-  negateRule,
-  orRule,
-} from "./Syntax/LogicalExpressionSyntax";
+import { andRule, negateRule, orRule } from "./Syntax/LogicalExpressionSyntax";
 
 // the order is important - tokens are applied from first to last
 const rules = [
   ...parenthesesRule,
   orRule,
   andRule,
-  negatedVariableRule,
+  negateRule,
   variableRule,
   blankRule,
 ];
-const rulesNegation = [negateRule, variableRule, blankRule];
-
-interface LeveledToken {
-  level: number;
-  token: Token;
-}
 
 export class LogicalExpressionParser {
   private lex: Lexer = new Lexer(rules);
-  private lexNegation: Lexer = new Lexer(rulesNegation);
-  private output: string = "";
 
   /**
    * Converts a logical expression to an expression in dimacs sat format
    * dimacs sat format: https://www.domagoj-babic.com/uploads/ResearchProjects/Spear/dimacs-cnf.pdf
    * @param logicalExpression {string} logical expression
    * @returns {string} formula in dimacs sat format of the input
```

Comment:
Is this `@returns` still correct? the type seems wrong and if i see this correctly, the meaning of the return value has changed, too